src/components/measureGroups/MeasureGroups.tsx

```diff
@@ -640,114 +640,141 @@ const MeasureGroups = () => {
             {activeTab === "stratification" && (
               <FormControl>
                 {formik.values.stratifications.map((strat, i) => (
-                  //Can be made into it's own component, later"
-                  <Row>
-                    <Col>
-                      <FieldLabel htmlFor="stratification-select">
-                        Stratification {i + 1}
-                      </FieldLabel>
-                      <TextField
-                        select
-                        id="stratification-select"
-                        label=""
-                        value={formik.values.stratifications[i].cqlDefinition}
-                        inputProps={{
-                          "data-testid": "stratification-select",
-                        }}
-                        onChange={(e) => {
-                          formik.setFieldValue(
-                            `stratifications[${i}]cqlDefinition`,
-                            e.target.value
-                          );
-                        }}
-                        InputLabelProps={{ shrink: false }}
-                        SelectProps={{
-                          native: true,
-                        }}
-                        name="type"
-                      >
-                        <option
-                          value=""
-                          data-testid="stratification-select-option"
+                  <div>
+                    {
+                      //Can be made into it's own component, later
+                    }
+                    <Row>
+                      <Col>
+                        <FieldLabel htmlFor="stratification-select">
+                          Stratification {i + 1}
+                        </FieldLabel>
+                        <TextField
+                          select
+                          id="stratification-select"
+                          label=""
+                          value={formik.values.stratifications[i].cqlDefinition}
+                          inputProps={{
+                            "data-testid": "stratification-select",
+                          }}
+                          onChange={(e) => {
+                            formik.setFieldValue(
+                              `stratifications[${i}]cqlDefinition`,
+                              e.target.value
+                            );
+                          }}
```

Comment:
You can avoid manual binding.
`onChange={formik.handleChange}` would be sufficient